ecosystem/sep-0006.md

```diff
@@ -769,16 +768,19 @@ The response body should be a JSON object with the following fields:
 | `account_id` | `G...` string | (optional) The account the user should send its token back to. This field can be omitted if the anchor cannot provide this information at the time of the request. |
 
 In this case, the wallet should query the [`/transaction`](#single-historical-transaction) endpoint to get this
-asynchonously. `memo_type` | string | (optional) Type of memo to attach to transaction, one of `text`, `id` or `hash`.
-`memo` | string | (optional) Value of memo to attach to transaction, for `hash` this should be base64-encoded. The
-anchor should use this memo to match the Stellar transaction with the database entry associated created to represent it.
-`id` | string | (optional) The anchor's ID for this withdrawal. The wallet will use this ID to query the
-[`/transaction`](#single-historical-transaction) endpoint to check status of the request. `eta` | int | (optional)
-Estimate of how long the withdrawal will take to credit in seconds. `min_amount` | float | (optional) Minimum amount of
-an asset that a user can withdraw. `max_amount` | float | (optional) Maximum amount of asset that a user can withdraw.
-`fee_fixed` | float | (optional) If there is a fee for withdraw. In units of the withdrawn asset. `fee_percent` | float
-| (optional) If there is a percent fee for withdraw. `extra_info` | object | (optional) Any additional data needed as an
-input for this withdraw, example: Bank Name.
```

Comment:
Is this related to the KYC change? Should we be deprecating this first before removing it?

Reply:
I think was a formatting issue that @Ifropc fixed below.

Reply:
Formatting issue!